galaxy/scripts/probedb.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+
+import argparse
+import psycopg2
+import socket
+import sys
+
+
+from datetime import datetime
+from time import sleep
+
+
+DEBUG=False
+# allowable seconds since db timestamp
+default_interval=60
+
+def dprint(msg):
+    if DEBUG:
+        print(msg)
+
+parser = argparse.ArgumentParser()
```

Comment:
It would be good to define a main function and move the code there, following this example: https://github.com/galaxyproject/galaxy/blob/6da7b1ade8866dd9048881c20e37815a79e50f89/scripts/api/fetch_to_library.py#L33
This is a more comprehensive sample:
https://github.com/galaxyproject/galaxy/blob/dev/scripts/cleanup_datasets/cleanup_datasets.py

Reply:
suggested change [implemented](https://github.com/luke-c-sargent/galaxy-helm/blob/10e20ed149970e0ab6bdca95192726cca26b4262/galaxy/scripts/probedb.py#L40)